staging/src/k8s.io/kubectl/pkg/cmd/version/version.go

```diff
@@ -36,10 +38,20 @@ import (
 
 // Version is a struct for version information
 type Version struct {
-	ClientVersion *apimachineryversion.Info `json:"clientVersion,omitempty" yaml:"clientVersion,omitempty"`
+	ClientVersion KubectlVersion            `json:"clientVersion,omitempty" yaml:"clientVersion,omitempty"`
```

Comment:
This is one of the changes I'm concerned might be problematic. If this is immutable, we could leave it as is and have the version command use a new, private struct instead.